### DISCUSSION TITLE
"starred a repository"

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#27416] [Other Feature Feedback, Questions, & Ideas] [answered]
This should be "starred a repository", it shouldnt be "published a discussion" right?

![Screenshot_20220728-102613.png](https://user-images.githubusercontent.com/94877880/181423891-02464f32-d793-4d86-9fe1-e88f3821b6cd.png)

**Comment by sawyerbivens** [comment]
I assume it means that in the discussions tab they created a discussion, like the one you created here. But it may be a bug I'm not sure why it would show the star icon. Who knows

**Reply by NitinIsFit** [reply]
I did see the repo, there isnt any discussion published by them

**Reply by sawyerbivens** [reply]
Hm then yea must be a bug, maybe someone at Github will see this and bring attention to it

**Reply by NitinIsFit** [reply]
![Screenshot_20220728-103533.png](https://user-images.githubusercontent.com/94877880/181424943-2ea31d6c-6ad8-400b-b219-24a7c3bca2bd.png)

This is a read discussion right? It have a "join discussion" button too, and we are able to react on them with emotes.

Earlier one shows that some user have "started a repo" but it still says "published a discussion"

**Reply by sawyerbivens** [reply]
Yea that would be a discussion

Have you tried fully restarting the Github app, maybe some kind of label text or something got mixed up I'm not really sure. My app seems to be normal

**Reply by NitinIsFit** [reply]
> Yea that would be a discussion
> 
> Have you tried fully restarting the Github app, maybe some kind of label text or something got mixed up I'm not really sure. My app seems to be normal

Yeah I did restart the app, and I'm so addicted to GitHub that i open it in every 10 minutes lmaoo.
Still the big is here.

**Reply by sawyerbivens** [reply]
Yea I'm not sure how that happens, if you're on the latest version of the Github app it should be fine but hopefully this gets seen by someone at Github to be fixed

**Reply by siminapasat** [reply]
👋 We're looking into it! 😊

**Reply by siminapasat** [reply]
Update: it's already fixed in our beta version, fix will be rolling out to the next app release 👍

**Reply by sawyerbivens** [reply]
> Update: it's already fixed in our beta version, fix will be rolling out to the next app release 👍

thats great!

**Reply by NitinIsFit** [reply]
> Update: it's already fixed in our beta version, fix will be rolling out to the next app release 👍 

Awesome to hear

**Comment by siminapasat** [comment, marked as answer]
Update: it's already fixed in our beta version, fix will be rolling out to the next app release 👍

**Comment by Karkatov** [comment]
I will mutually add your repository to favorites, here is mine: https://github.com/Karkatov/TaskManager
